lib/Ride.dart

```diff
@@ -12,33 +12,30 @@ class Ride {
   final String endLocation;
   final DateTime startTime;
   final DateTime endTime;
-  final String riderId;
-  // can be null
-  final String driverId;
+  final Rider rider;
 
   Ride({
     this.id,
     this.type,
     this.startLocation,
     this.endLocation,
-    this.riderId,
-    this.driverId,
```

Comment:
Are you removing driver because a driver would never need the information on the driver since it is them? Because that might not be the case when we start showing other drivers' schedules.

Reply:
I thought about this but thought maybe we can leave it for when the designs for that are finalized, especially since I'm not 100% sure what info we'll need for each driver, like if we just need ID and name or if we'll also need phone number, etc. Should I add it back and just make a Driver class, or wait on it?

Reply:
I'd say it's fine to remove it for now. Like you say, we can always change things up when the designs are finalized.